services/pkg/lsyncd/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM  ghcr.io/void-linux/void-glibc:latest
+
+RUN xbps-install -Sy lsyncd tini && rm -rf /var/cache/xbps
+
+ENTRYPOINT ["/usr/bin/tini", "--"]
+CMD ["/usr/bin/lsyncd", "-nodaemon", "/etc/lsyncd.conf"]
```

Comment:
Consider changing to /local/lsyncd.conf to avoid the need to do bind mount shuffling at runtime.